### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by mscherer**

### CODE REVIEW

**Review by anishasthana — APPROVED** [review]
/lgtm

File: docs/setup_cd_pipeline.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Configuration files allows user assign details about the build requirements and
 
 ```yaml
 build:
-  build-stratergy: Dockerfile # Allowed values: Source, Dockerfile, Containerfile (default: Source)
```
Comment (tumido) [inline docs/setup_cd_pipeline.md:56]: Unfortunately, this has to stay as is see here: https://github.com/AICoE/aicoe-ci/issues/88

**Review by tumido — CHANGES_REQUESTED** [review]

File: docs/setup_cd_pipeline.md (hunk 1 above)
Comment (harshad16) [reply docs/setup_cd_pipeline.md:56]: Thanks for pointing this out in-here.

**Comment by harshad16** [comment]
Thanks for the fix
we have updated the services accordingly.

**Pull request merged**